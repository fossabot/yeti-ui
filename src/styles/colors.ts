export const PRIMARY = '#1779ba';
export const SECONDARY = '#767676';
export const SUCCESS = '#3adb76';
export const WARNING = '#ffae00';
export const ALERT = '#cc4b37';

export const LIGHT_GRAY = '#e6e6e6';
export const MEDIUM_GRAY = '#cacaca';
export const DARK_GRAY = '#8a8a8a';
export const BLACK = '#0a0a0a';
export const WHITE = '#fefefe';
