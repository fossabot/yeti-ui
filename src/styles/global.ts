export enum FOUNDATION_PALETTE {
  PRIMARY = '#1779ba',
  SECONDARY = '#767676',
  SUCCESS = '#3adb76',
  WARNING = '#ffae00',
  ALERT = '#cc4b37',
}

export const FONT_SIZE = '100%';
export const WIDTH = 'rem-calc(1200)';
export const LINEHEIGHT = '1.5';
export const LIGHT_GRAY = '#e6e6e6';
export const MEDIUM_GRAY = '#cacaca';
export const DARK_GRAY = '#8a8a8a';
export const BLACK = '#0a0a0a';
export const WHITE = '#fefefe';
export const BODY_BACKGROUND = WHITE;
export const BODY_FONT_COLOR = BLACK;
export const BODY_FONT_FAMILY = '\'Helvetica Neue\', Helvetica, Roboto, Arial, sans-serif';
export const BODY_ANTIALIASED = true;
export const MARGIN = '1rem';
export const PADDING = '1rem';
export const POSITION = '1rem';
export const WEIGHT_NORMAL = 'normal';
export const WEIGHT_BOLD = 'bold';
export const RADIUS = '0';
export const MENU_PADDING = '0.7rem 1rem';
export const MENU_NESTED_MARGIN = '1rem';
export const TEXT_DIRECTION = 'ltr';
export const FLEXBOX = 'true';
export const PROTOTYPE_BREAKPOINTS = false;
export const BUTTON_CURSOR = 'auto';
export const LEFT = 'left';
export const RIGHT = 'right';
export const COLOR_PICK_CONTRAST_TOLERANCE = 0;
