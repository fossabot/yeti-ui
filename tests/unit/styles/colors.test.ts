import * as colors from '../../../src/styles/colors';

describe('src/styles/colors.ts', () => {

  test('colors should match snapshot', () => {
    expect(colors).toMatchSnapshot();
  });
});
