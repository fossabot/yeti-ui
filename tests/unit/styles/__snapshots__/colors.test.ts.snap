// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/styles/colors.ts colors should match snapshot 1`] = `
Object {
  "ALERT": "#cc4b37",
  "BLACK": "#0a0a0a",
  "DARK_GRAY": "#8a8a8a",
  "LIGHT_GRAY": "#e6e6e6",
  "MEDIUM_GRAY": "#cacaca",
  "PRIMARY": "#1779ba",
  "SECONDARY": "#767676",
  "SUCCESS": "#3adb76",
  "WARNING": "#ffae00",
  "WHITE": "#fefefe",
}
`;
